### PR TITLE
Add a disclaimer message for all `docker compose` commands

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@gmail.com>

**What I did**

**Related issue**
https://github.com/docker/dev-tooling-team/issues/244

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

**Comment by gtardif** [comment]
```
$ ./bin/docker compose down -f tests/composefiles/demo_multi_port.yaml
the new 'docker compose' command is currently experimental. To provide feedback or request new features please open issues on https://github.com/docker/compose-cli
[+] Running 4/4
 ⠿ Container composefiles_web_1    Removed                                                                                                                 0.4s
 ⠿ Container composefiles_db_1     Removed                                                                                                                 0.4s
 ⠿ Container composefiles_words_1  Removed                                                                                                                 0.7s
 ⠿ Network "composefiles_default"  Removed                                                                                                                 0.2s
```

File: cli/cmd/compose/compose.go [hunk 1]
```diff
@@ -72,6 +74,10 @@ func Command(contextType string) *cobra.Command {
 	command := &cobra.Command{
 		Short: "Docker Compose",
 		Use:   "compose",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println("the new 'docker compose' command is currently experimental. To provide feedback or request new features please open issues at https://github.com/docker/compose-cli")
```
Comment (rumpl) [inline cli/cmd/compose/compose.go:78]: Nit
```suggestion
			fmt.Println("The new 'docker compose' command is currently experimental. To provide feedback or request new features please open issues at https://github.com/docker/compose-cli")
```

**Review by rumpl — APPROVED** [review]

**Review by nebuk89 — APPROVED** [review]
Yep, let's ship this and see if people are still confused. If they are we will add something to make it clearer that this is different from docker-compose

**Comment by lukasmrtvy** [comment]
Guys? Is possible to hide this disclaimer somehow? I need to parse output from compose-cli...

**Pull request merged**